homedecor_common/registration.lua

```diff
@@ -46,9 +46,9 @@ function homedecor.register(name, original_def)
 	if expand then
 		-- dissallow rotating only half the expanded node by default
 		-- unless we know better
-		def.on_rotate = def.on_rotate
-			or (minetest.get_modpath("screwdriver") and (def.mesh and expand.top and screwdriver.rotate_simple)
-			or screwdriver.disallow) or nil
+		def.on_rotate = def.on_rotate or
+		(minetest.get_modpath("screwdriver") and (def.mesh and expand.top and screwdriver.rotate_simple) or
+		minetest.get_modpath("screwdriver") and screwdriver.disallow) or nil
```

Comment:
```lua
def.on_rotate = def.onrate or (
    minetest.get_modpath("screwdriver") and (
        (def.mesh and expand.top and screwdriver.rotate_simple) or
        screwdriver.disallow
    )
)

```

Reply:
out of scope for the pr

Reply:
"making the code legible" is in the scope of every PR

Reply:
of parts they touch

Reply:
this is minor enough ill add it

Reply:
https://github.com/mt-mods/homedecor_modpack/pull/28/commits/ee77cf77751e74b29f7120c79a8996f9b0efeeb4 fixed